### ISSUE TITLE
Some characters be encoded into bad byte sequence in browser

### ISSUE THREAD

**Issue by kevinat** [rse/encodr#1]
```js
const CBOR = new Encodr("cbor")
console.log(CBOR.encode('a￥a'));

// returns [101, 97, 244, 137, 145, 161], but it should be [101, 97, 239, 191, 165, 97]
// when run this code in nodeJS, correct result returns.

```

**Comment by rse** [comment]
You are right: in the browser the encoding is broken. I've still no clue why, but I'll try to find out.

**Comment by rse** [comment]
I've created a stand-alone sample and it looks the encoding of the underlying "cbor" module is broken in browser environments. I've [reported the problem](https://github.com/hildjj/node-cbor/issues/142) to the upstream author of the "cbor" module. Perhaps he can shed some light on the problem.

**Comment by rse** [comment]
Now fixed with version 1.3.0
